domains/authorization/Authorization.API/API/Authorization/[Features]/Internal/GetServiceProviderTermsForOrganizationQuery.cs

```diff
@@ -0,0 +1,49 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Authorization.Exceptions;
+using API.Repository;
+using EnergyOrigin.Domain.ValueObjects;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Authorization._Features_.Internal;
+
+public record GetServiceProviderTermsForOrganizationQuery(string OrgCvr) : IRequest<bool>;
```

Comment:
We should use the OrgId since it's our primary key.

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1605/commits/4950dc0e89bc222374cf16496ee20b9da9dfd2af